packages/components/src/components/link-tile/link-tile-styles.ts

```diff
@@ -149,30 +151,25 @@ export const getComponentCss = (
           isCompact
             ? {
                 alignItems: 'center',
-                gridTemplateColumns: `auto ${pxToRemWithUnit(24)}`,
+                gridTemplateColumns: 'auto 24px',
                 gridTemplateRows: 'auto',
                 ...(isTopAligned ? { top: 0 } : { bottom: 0 }),
               }
             : { gridTemplateRows: 'auto auto', gridTemplateColumns: 'auto' }
         )
       ),
     },
-
     'link-pure': buildResponsiveStyles(compact, (isCompact: boolean) => ({
       display: isCompact ? 'inline-block' : 'none',
     })),
     link: {
-      minHeight: '3rem',
-      ...buildResponsiveStyles(compact, (isCompact: boolean) => ({ display: isCompact ? 'none' : 'inline-flex' })),
+      minHeight: '54px', // prevent content shift
+      ...buildResponsiveStyles(compact, (isCompact: boolean) => ({ display: isCompact ? 'none' : 'inline-block' })),
     },
-
-    // Due to position absolut on .content, position fixed is used to expand the clickable area of the anchor onto the whole link-tile
-    anchor: {
-      '&::after': {
-        content: '""',
-        position: 'fixed',
-        ...getInsetJssStyle(),
-      },
+    'link-overlay': {
```

Comment:
Might be good to comment for what problem or limitation this is the solution for.

Reply:
Done